plugin/salve.vim

```diff
@@ -220,40 +219,27 @@ function! s:projectionist_detect() abort
   let projections = {}
 
   let proj = {'type': 'test', 'alternate': map(copy(main), 'v:val."/{}.clj"')}
-  let proj = {'type': 'test', 'alternate': map(copy(main), 'v:val."/{}.cljc"')}
```

Comment:
This is the real problem, I believe. Changing it to something like `map(copy(main), 'v:val."/{}.cljc"') + map(copy(main), 'v:val."/{}.clj"')` should fix it without throwing out the baby with the bathwater.